.circleci/config.yml

```
@@ -610,14 +610,14 @@ jobs:
             END_TIME=$(date +%s)
 
             if << pipeline.parameters.randomize-aws-credentials >> ; then
-              echo "export TINYBIRD_WORKFLOW=tests_circleci_ma_mr" >> $BASH_ENV
+              export TINYBIRD_WORKFLOW=tests_circleci_ma_mr
```

```suggestion
              TINYBIRD_WORKFLOW=tests_circleci_ma_mr
```
Couldn't we just do this like we do for the timestamps above?